frontend/public/style/_icons.scss

```diff
@@ -4,4 +4,5 @@
 
 .co-icon-space-r {
   margin-right: 0.25em;
+  margin-left: 2px;
```

Comment:
We wouldn't want to modify this class, since it would effect a lot of other instances where it's used. 

I believe the bug issue is isolated in the code https://github.com/openshift/console/blob/master/frontend/public/components/dashboard/dashboards-page/cluster-dashboard/details-card.tsx#L79-L82

I would rather add a `className="co-icon-space-l"` to the `Link` component.

```
<Link to="/settings/cluster?showVersions" className="co-icon-space-l">
                <BlueArrowCircleUpIcon className="co-icon-space-r" />
                {t('public~Update cluster')}
              </Link>```